src/components/Nav/Nav.tsx

```diff
@@ -15,8 +22,22 @@ export function NavbarScreen() {
         </Link>
         <p className="ml-2 text-2xl">CapyFile</p>
         <div className="flex flex-grow justify-end gap-4">
-          <Button text="Login" to="/login" />
-          <Button text="Register" to="/register" />
+          <div className="lg:hidden">
+            <button aria-label="Open menu" onClick={toggleMobileMenu}>
+              <Menu size={54} />
+            </button>
+          </div>
+          {showMobileMenu && (
+            <div className="absolute left-0 right-0 top-32 flex -translate-y-1/2 transform items-center justify-center bg-white p-2 lg:hidden">
```

Comment:
1. Use the `md` breakpoint instead of `lg` to hide the buttons. 
2. Try to make a column layout as the shown in the following image. It would be nice if it takes the full screen height: 

![image](https://github.com/hawks-atlanta/frontend-react/assets/62714297/4dde57ad-9742-4ca4-87ca-56fb0cea6aa9)